lookout/style/format/benchmarks/release_v0.1/30-seconds-of-code.quality_report.md

```diff
@@ -0,0 +1,68 @@
+# Quality report
+
+### Classification report:
+| class | precision | recall | f1-score | support |
+|----:|:-----|:-------|:-----|:-----|
+| `␣` | `0.95` | `1.00` | `0.98` | `11409` |
```

Comment:
Would be nice to align table cells so that they are human-readable in plain text

Reply:
Can be in a separate PR